scripts/flash/list-market-orders.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var chalk = require("chalk");
+var async = require("async");
+var displayTime = require("./display-time");
+
+function getMarketOrdersInternal(augur, marketIds, universe, auth, callback) {
+  augur.api.Controller.getTimestamp(function (err, timestamp) {
+    augur.markets.getMarkets({ universe: universe, sortBy: "endDate", isSortDescending: true }, function (err, marketIds) {
+      if (!marketIds || marketIds.length === 0) {
+        console.log(chalk.red("No markets available"));
+        callback("No Markets");
+      }
+      augur.markets.getMarketsInfo({ marketIds: marketIds }, function (err, marketInfos) {
+        if (err) {
+          console.log(chalk.red("Error "), chalk.red(err));
+          return callback(err);
+        }
+        if (!marketInfos || !Array.isArray(marketInfos) || !marketInfos.length) {
+          return callback("No Market Info");
+        }
+        var market = marketInfos[0];
+        var marketId = market.id;
+        var outcomes = [0, 1];
+        if (market.marketType === "category") {
+          outcomes = new Array(market.numOutcomes);
```

Comment:
`market.numOutcomes` will always have the right number, you could just do this one line every time instead of switching when it is `category`